### PR TITLE
Klaus branch

### PR DESCRIPTION
**Pull request by KHardt**
## Link to Ticket
Closes # [3](https://github.com/NSS-Therapeutic-Raccoons/BangazonAPI/issues/3)

## Description of Proposed Changes
Proposed changes are for ticket 3 regarding Product Types. Controller and Model created for Product Types. 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout KlausBranch

```

## Impacted Areas in Application

List general components of the application that this PR will affect:

* ProductType Controller and Model being added.
* Should not affect any other areas of application yet. There are no foreign keys in this table. However will be referenced later. 

## Mentions To Review

@jeremiah3643, @askingalot, @ricky-bruner, @thatmikeparrish    

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.

### CODE REVIEW

**Review by ricky-bruner — DISMISSED** [review]
looking at files, documentation still needed.

**Review by ricky-bruner — CHANGES_REQUESTED** [review]
On your first Get in your API controller, please remove the string q parameter, as the ticket does not request this specificity. We can add these things later once overall project goals are met.

Your Model also need to remove the list, as it does not call for a product types list in the ticket.

**Review by ricky-bruner — APPROVED** [review]

**Comment by ricky-bruner** [comment]
Changes look great, All controller verbs test perfectly in postman. Great job Klaus!

**Comment by thatmikeparrish** [comment]
heyo

**Review by thatmikeparrish — APPROVED** [review]
Pulled and tested, looks good

File: BangazonAPI/Models/ProductTypes.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+Author: Klaus Hardt
+Purpose: Model for Product Types including gets/sets for all db columns and List. Also Ricky sucks.
+*/
+using System.Collections.Generic;
+
+namespace BangazonAPI.Data
+{
+    public class ProductTypes
```
Comment (askingalot) [inline BangazonAPI/Models/ProductTypes.cs]: This should be `ProductType` not `ProductTypes`

**Review by askingalot — CHANGES_REQUESTED** [review]

File: BangazonAPI/Models/ProductTypes.cs (hunk 1 above)
Comment (ricky-bruner) [reply BangazonAPI/Models/ProductTypes.cs]: woooooooow

**Review by askingalot — APPROVED** [review]
Looks good

**Comment by ricky-bruner** [comment]
THANKS ANDY

**Pull request merged**